autointent/generation/utterances/basic/chat_template.py

```diff
@@ -0,0 +1,131 @@
+"""Chat template for evolution augmentation via abstractization."""
+
+import random
+from abc import ABC, abstractmethod
+from typing import ClassVar
+
+from autointent import Dataset
+from autointent.generation.utterances.schemas import Message, Role
+from autointent.schemas import Intent
+
+
+class BaseSynthesizer(ABC):
+    """Base class."""
+
+    @abstractmethod
+    def __call__(self, intent_data: Intent, n_examples: int) -> list[Message]:
+        """Generate examples for this intent."""
+
+
+class SynthesizerChatTemplate(BaseSynthesizer):
+    """Chat template for generating additional examples for a given intent class."""
+
+    _messages: ClassVar[list[Message]] = [
+        Message(
+            role=Role.USER,
+            content=(
+                "You will be provided with a set of example utterances and the name "
+                "of the common topic (intent name) of these utterances. "
+                "Your task is to generate more examples that fit within the same intent name.\n\n"
+                "Note:\n"
+                "- You can generate similar utterances with only slot values changed\n"
+                "- You can generate completely different utterance from the same intent name\n"
+                "- Intent name can be missed, then you should infer from example utterances only\n"
+                "- Example utterances can be missed, then you should infer from intent name only\n"
+                "{extra_instructions}\n\n"
```

Comment:
```suggestion
                "{extra_instructions}\n\n"
```
Кажется тут часть темлейта

Reply:
не, это норм, в конструкторе эта часть заполняется